test/e2e/config/docker.yaml

```diff
@@ -254,10 +254,10 @@ variables:
   # allowing the same e2e config file to be re-used in different Prow jobs e.g. each one with a K8s version permutation.
   # The following Kubernetes versions should be the latest versions with already published kindest/node images.
   # This avoids building node images in the default case which improves the test duration significantly.
-  KUBERNETES_VERSION_MANAGEMENT: "v1.25.3"
-  KUBERNETES_VERSION: "v1.25.3"
-  KUBERNETES_VERSION_UPGRADE_FROM: "v1.24.7"
-  KUBERNETES_VERSION_UPGRADE_TO: "v1.25.3"
+  KUBERNETES_VERSION_MANAGEMENT: "v1.26.0"
+  KUBERNETES_VERSION: "v1.26.0"
+  KUBERNETES_VERSION_UPGRADE_FROM: "v1.25.3"
+  KUBERNETES_VERSION_UPGRADE_TO: "v1.26.0"
   ETCD_VERSION_UPGRADE_TO: "3.5.4-0"
```

Comment:
Looks like kubeadm is now using `3.5.6-0`, let's update the var here accordingly

(https://github.com/kubernetes/kubernetes/blob/v1.26.0/cmd/kubeadm/app/constants/constants.go#L312)